packages/pluggable-widgets-tools/CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 ## [Unreleased]
 
+### Added
+
+- We added the property type AttributeMetaData which exposes useful metadata for filtering and sorting datasources.
```

Comment:
I am not sure we should expose it in the changelog as we are not going public with this feature just yet

Reply:
I can go either way on this. On the one hand it is nice to keep this feature close to the chest for now. On the other hand I enjoy the transparency of what changed in this particular version of the package. The generated experimental types are annotated with `@beta` so any user of it is warned...

As it is a single commit it is easily dropped.